### DISCUSSION TITLE
hosting with other providers

### DISCUSSION THREAD

**Discussion by hellking101** [saran-surya/email_auth_node#66] [Q&A] [answered]
Hi thanks a lot for this package and the amazing work you have done for making it easier to send OTP to emails then Firebase (looking at you google make this a thing already dame it).
i want to know how can I host this package in Azure if that is even possible thanks again for your hard work

**Comment by saran-surya** [comment, marked as answer]
Not sure of FireBase's SDK part. They are part of a bigger picture.

Coming back to the topic, If you have access to NODE JS SDK in Azure you can host this the same way. Just that you can override the routes, This place can be a very good start : https://learn.microsoft.com/en-in/azure/developer/javascript/
